wordpress/theme/lib/editor/blocks-whitelist.tsx

```diff
@@ -22,6 +22,8 @@ const BlocksWhitelist = (): JSX.Element | null => {
 	const whiteListBlocks = useCallback((postType: PostType) => {
 		const blockTypes = getBlockTypes();
 		blockTypes.forEach((blockType) => {
+			if (blockType.name === 'core/block') return; // Skip core/block as it's used for pattern/reusable blocks
+
 			if (
```

Comment:
Just to be sure, this does allows `core/block` in the editor ? Or does it removes it

Reply:
@kuuak Yep it allows it regardless of the post type (because we check according to the `postTypes` setting of the block, but we don't set any `postTypes` for this block.

Reply:
Perfect then 👏